Userland/Libraries/LibC/sys/arch/regs.h

```diff
@@ -12,4 +12,6 @@
 #    include "i386/regs.h"
 #elif ARCH(X86_64)
 #    include "x86_64/regs.h"
+#elif ARCH(AARCH64)
+#    include "aarch64/regs.h" // FIXME
```

Comment:
Is this still a FIXME if you implemented it in this commit? One option might be to leave this change to arch/regs.h and the actual implementation of arch/aarch64/regs.h for a second commit, and title the commit named `Everywhere: Single import place for x86 and arm64 registers` as: `LibC+Everywhere: Consistently use sys/arch/regs.h helper file`

Reply:
awww! 🫢 iʼll take the approach for the change too!